Documentation/project-docs/developer-guide.md

```diff
@@ -196,28 +195,28 @@ msbuild /t:Test "/p:XunitOptions=-class Test.ClassUnderTests"
 
 #### What is TestTFM and what possible values can it have
 
-`TestTFM` is the Framework that we will use to run your tests on. The same test assembly can be used to run tests on two different frameworks, for example, AssemblyWithMyTests.dll can be used to run your tests in TFM `A` and `B` as long as framworks `A` and `B` provide/support the NetStandard surface area needed for the test assembly to compile. For this reason, when you write your tests, you might want to run them against different frameworks, and the next section will point out how to do that.
+`TestTFM` is the Framework that we will use to run your tests on. The same test assembly can be used to run tests on two different frameworks, for example, AssemblyWithMyTests.dll can be used to run your tests in TFM `A` and `B` as long as frameworks `A` and `B` provide/support the NetStandard surface area needed for the test assembly to compile. For this reason, when you write your tests, you might want to run them against different frameworks, and the next section will point out how to do that.
 
 Some of the possible values for `TestTFM` are:
 
-_**`netcoreapp1.1` or `netcoreapp1.0`**_
+- _**`netcoreapp1.1` or `netcoreapp1.0`**_
 NetStandard implementations that run in CoreCLR.
 
-_**`netcore50` or `uap101aot`**_
+- _**`netcore50` or `uap101aot`**_
 NetStandard implementations for UWP.
 
--**`net46` or `net462` or `net463`**-
+- _**`net46` or `net462` or `net463`**_
 NetStandard implementations for Desktop.
 
 #### Running tests in a different TFM
 
-Each test project corresponds to a test .builds file. There are some tests that might be OS specific, or might be testing API that is available only on some TFMs, which is what this tests.builds files are for. By default, we will build all of these different configurations always, but we will only execute the tests on one TFM. By default, our `TestTFM` is set to `netcoreapp1.1` [here](https://github.com/dotnet/corefx/blob/master/dir.props#L495) which means that we will run tests for the configurations that either:
-- Have `netcoreapp1.1` inside their `<TestTFMs>...</TestTFMs>` clause on the .builds files, or
-- Don't have a `<TestTFMs>...</TestTFMs>` metadata in the .builds file at all because we default it to netcoreapp1.1 [here](https://github.com/dotnet/corefx/blob/master/dir.traversal.targets#L146-L147)
+Each test project corresponds to a test .builds file. There are some tests that might be OS-specific, or might be testing an API that is available only on some TFMs, so the tests.builds specifies the valid configurations. By default, we will build all of these different configurations always, but we will only execute the tests on one TFM. By default, our `TestTFM` is set to `netcoreapp1.1` [here](https://github.com/dotnet/corefx/blob/80ab4804aeb7bed16e64b988c7460c705fddd5cc/dir.props#L529) which means that we will run tests for the `Project`s in the .builds file that either:
```

Comment:
I changed it to point to a specific commit because it's pointing to a line number within the file, and the intended line number can change as the file is edited.  In fact, currently the link is pointing to the wrong line (https://github.com/dotnet/corefx/blob/master/dir.props#L495).

I agree that it's useful to have it point to the latest master version of the file, though, so what do you think about just pointing to the file instead of pointing to the line number?  The downside is that people would have to find where `DefaultTestTFM` is set within the file.